### PR TITLE
update automating-pipelines guide with auto-materialize policies

### PR DESCRIPTION
**Pull request by sryza**

### CODE REVIEW

File: docs/content/guides/dagster/automating-pipelines.mdx [hunk 1]
```diff
@@ -24,11 +24,11 @@ When thinking about automating your data pipelines, it’s helpful to think thro
...
+You want computation to be triggered based on the time. For example, you want some data to be updated every day by a certain time to refresh a dashboard, or you want your data warehouse to have the most recent information for the team to work with. This is one of the most traditional cases when building a pipeline. Let's go into how to do this with Dagster!
 
 ### I want a job to start at a specific time
 
-In Dagster, we have [basic scheduling](/concepts/partitions-schedules-sensors/schedules#basic-schedules) capabilities, which means that you say how often and when you want a job to run, and it will run. This can mean daily, weekly, or hourly. Dagster also supports any type of [cron scheduling](https://en.wikipedia.org/wiki/Cron).
+Dagster, offers [basic scheduling](/concepts/partitions-schedules-sensors/schedules#basic-schedules) capabilities that allow you to specify how often and when you want a job to run. This can mean daily, weekly, or hourly. Dagster also supports any type of [cron scheduling](https://en.wikipedia.org/wiki/Cron).
```
Comment (odette-elementl) [inline docs/content/guides/dagster/automating-pipelines.mdx:31]: not sure we need comma after Dagster,

File: docs/content/guides/dagster/automating-pipelines.mdx [hunk 2]
```diff
@@ -55,46 +55,54 @@ def configurable_job_schedule(context: ScheduleEvaluationContext):
     )
 ```
 
-### I want my data to be updated at different cadences
+### I want specify how fresh each asset needs to be and let Dagster figure out when to run them
```
Comment (odette-elementl) [inline docs/content/guides/dagster/automating-pipelines.mdx:58]: Do we need to say what fresh means?

**Review by odette-elementl — APPROVED** [review]

File: docs/content/guides/dagster/automating-pipelines.mdx (hunk 2 above)
Comment (sryza) [reply docs/content/guides/dagster/automating-pipelines.mdx:58]: I think we can get away without it here

Comment (sryza) [reply docs/content/guides/dagster/automating-pipelines.mdx:58]: given that we explain it in other terms in the text below

**Pull request merged**